Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -117,7 +117,17 @@ function Cesium3DTilesInspector(container, scene) {
   tilesetPanelContents.appendChild(createCheckbox("Enable Picking", "picking"));
 
   displayPanelContents.appendChild(createCheckbox("Colorize", "colorize"));
-  displayPanelContents.appendChild(createCheckbox("Wireframe", "wireframe"));
+  displayPanelContents.appendChild(
+    createCheckbox("Wireframe", "wireframe", "hasEnabledWireframe")
+  );
+
+  //Create warning text when the Wireframe checkbox is disabled
+  const warningText = document.createElement("p");
+  warningText.setAttribute("data-bind", `visible: !hasEnabledWireframe`);
+  warningText.setAttribute("style", "margin: 5px 0 0 0; padding: 0 0 0 20px");
```

Comment:
Also since the text is a note, could we style it differently (maybe a slightly different color or text size)?

![image](https://user-images.githubusercontent.com/8422414/191540206-19e0076d-83a2-4a87-8f7e-1e8c88c1b3ee.png)